Depth_First_Search/Depth_First_Search.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import "fmt"
+
+type set map[int]bool
+
+// Define Graph structure
+
+type Graph struct {
+
+    adjList map[int]set
+}
+
+// Create a graph and initialize its adjacency list, and return a pointer to it.
+
+func createGraph() *Graph {
+  
+    var g = Graph{}
+    g.adjList = make(map[int]set)
+    return &g
+}
+
+// Add an edge between the two given nodes.If a node does not exist in the graph yet, add it.
+
+func (g *Graph) addEdge(node1 int, node2 int) {
+  
+  // Check if node1 is already in the graph
```

Comment:
Hey, you don't need to add a blank line at all places. You can give a blank line after one if block